pkg/controller/job/job_controller.go

```diff
@@ -843,6 +843,7 @@ func (jm *Controller) manageJob(job *batch.Job, activePods []*v1.Pod, succeeded
 					if completionIndex != unknownCompletionIndex {
 						template = podTemplate.DeepCopy()
 						addCompletionIndexAnnotation(template, completionIndex)
+						addStableHostname(template, job.Name, completionIndex)
```

Comment:
why not just put the code here? The annotation required more lines, but this is one-liner.

Reply:
thought of the uniform of the code. I'll modify to one-line if it is better